includes/bootstrap.php

```diff
@@ -172,6 +172,19 @@ function initialize_features(): void {
 		$loader   = new Feature_Loader( $registry );
 		$loader->register_default_features();
 		$loader->initialize_features();
+
+		add_action(
+			'wp_abilities_api_categories_init',
+			static function () {
+				wp_register_ability_category(
+					'ai-experiments',
```

Comment:
What purpose will this category serve?

IMO categories should be semantic, since there can only be one, so if this is a catch-all group for any abilities we register in the plugin, then id avoid against this. We already get the same thing from the (required) ability name prefix.

If we need an identifier, and ability prefix isn't enough, then I recommend `WP_Ability::$metadata['experimental'] = true;`

Reply:
My understanding is that every Ability has to have a category associated with it. So I took the approach for now of registering this generic category that all the Abilities we end up with can utilize. Open to better approaches though, if for instance we want more specific categories

Reply:
> My understanding is that every Ability has to have a category associated with it.

Yeah, it's a chicken-egg situation. We wanted to ship a Category API to avoid a generic `uncategorized` scenario by adding it later, but until we establish what those categories should be, we're essentially doing the same generic bucketing 😅

I don't think a premature part of the API should hold up this PR, so if the other folks here can't think of something yet (in all fairness, ideal taxonomies form holistically), then I say we proceed as is and add a comment saying that this is a temporary catch-all and likely to change.

(related: https://wordpress.slack.com/archives/C08TJ8BPULS/p1762568388167959 )